Makefile.common

```diff
@@ -129,6 +129,7 @@ LDFLAGS_WHOLE_ARCHIVE_END ?= $(LDFLAGS_PREFIX)--no-whole-archive
 ### right now we should have only target-necessary flags, save them for exporting (for building ports) ###
 EXPORT_CFLAGS := $(CFLAGS)
 EXPORT_CXXFLAGS := $(CXXFLAGS)
+EXPORT_CPPFLAGS := $(CPPFLAGS)
```

Comment:
Why is `CPPFLAGS` needed? It leaks phoenix-specific flags like `NOMMU`

Reply:
Related: https://github.com/phoenix-rtos/phoenix-rtos-ports/pull/68

Reply:
It is not needed after all my bad.